.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug Report
+about: Tell us how Flask-RESTPlus is broken
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+### ***** **BEFORE LOGGING AN ISSUE** *****
+
+- Is this something you can **debug and fix**? Send a pull request! Bug fixes and documentation fixes are welcome.
+- Please check if a similar issue already exists or has been closed before. Seriously, nobody here is getting paid. Help us out and take five minutes to make sure you aren't submitting a duplicate.
```

Comment:
We could add a link to CONTRIBUTING.md as well

Reply:
I added a link to the contributing guidelines doc in commit 2a12fc6